client/dashboard/me/tax-details/index.tsx

```diff
@@ -1,11 +1,167 @@
+import { useResetSupportInteraction } from '@automattic/help-center/src/hooks/use-reset-support-interaction';
+import {
+	Card,
+	CardBody,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+} from '@wordpress/components';
+import { useDispatch as useDataStoreDispatch, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import { useAnalytics } from '../../app/analytics';
+import { useTaxName } from '../../app/hooks/use-country-list';
+import { useGeoLocationQuery } from '../../app/queries/geolocation';
+import InlineSupportLink from '../../components/inline-support-link';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import { localizeUrl } from './localize-url';
+import useRecordUserTaxEvents from './use-record-user-tax-events';
+import useUserTaxDetails from './use-user-tax-details';
+import UserTaxForm from './user-tax-form';
+import './style.scss';
+
+const HELP_CENTER_STORE = 'automattic/help-center';
+
+export default function UserTaxInfoPage() {
+	const { recordTracksEvent } = useAnalytics();
+	const translate = useTranslate();
+	const { data: geoData } = useGeoLocationQuery();
+	const { fetchError, userTaxDetails } = useUserTaxDetails();
+	const taxName = useTaxName(
+		userTaxDetails.country ?? userTaxDetails.country ?? geoData?.country_short ?? 'GB'
+	);
+	const resetSupportInteraction = useResetSupportInteraction();
+
+	const { setShowHelpCenter, setNavigateToRoute } = useDataStoreDispatch( HELP_CENTER_STORE );
```

Comment:
Could we please use the `useHelpCenter()` hook instead please? It ensures the help center gets async loaded. Same thing for the `useResetSupportInteraction()` hook; you might need to add that capability to the `useHelpCenter()` hook.

https://github.com/Automattic/wp-calypso/blob/e75f49d6168dc0fde8470da6c9d057baec39a39b/client/dashboard/app/help-center/index.ts#L14

Reply:
Changed in e51e39f.